Frontend.Tests/ServicesTests/TaskListServiceTests.cs

```diff
@@ -59,6 +59,84 @@ public void GivenAProjectID_PutsTheProjectReferenceInThePage()
             Assert.Equal(ProjectReference, _index.ProjectReference);
         }
 
+        public class LegalRequirementsStatusTest : TaskListServiceTests
+        {
+            public static IEnumerable<object[]> LegalRequirementsCompleted()
+            {
+                yield return new object[]
+                {
+                    new TransferLegalRequirements()
+                    {
+                        IsCompleted = true
+                    }
+                };
+                yield return new object[]
+                {
+                    new TransferLegalRequirements()
+                    {
+                        IsCompleted = true
+                    }
+                };
+            }
+
+            [Theory]
+            [MemberData(nameof(LegalRequirementsCompleted))]
+            public void GivenLegalRequirementsCompleted_StatusCompleted(TransferLegalRequirements info)
+            {
+                FoundProjectFromRepo.LegalRequirements = info;
+                _subject.BuildTaskListStatuses(_index);
+                Assert.Equal(ProjectStatuses.Completed, _index.LegalRequirementsStatus);
+            }
+
+            public static IEnumerable<object[]> LegalRequirementsInProgress()
```

Comment:
As a general convention I've bene putting any putting any non test methods at the bottom of the file. In my view it makes it a bit more easier to find any helper methods. But not precious about it. 
Maybe this method should be private.

Reply:
No problem, I've moved to the bottom of the file. They'll stay public though because the MemberData attribute requires them to be.